packages/grafana-ui/src/components/TimePicker/TimeRangeInput.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, FormEvent, useState } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme, TimeRange, TimeZone } from '@grafana/data';
+import { useStyles, useTheme } from '../../themes/ThemeContext';
+import { ClickOutsideWrapper } from '../ClickOutsideWrapper/ClickOutsideWrapper';
+import { Icon } from '../Icon/Icon';
+import { getInputStyles } from '../Input/Input';
+import { getFocusStyle } from '../Forms/commonStyles';
+import { TimePickerButtonLabel } from './TimeRangePicker';
+import { TimePickerContent } from './TimeRangePicker/TimePickerContent';
+import { otherOptions, quickOptions } from './rangeOptions';
+
+export interface Props {
+  value: TimeRange;
+  timeZone?: TimeZone;
+  onChange: (timeRange: TimeRange) => void;
+  onChangeTimeZone: (timeZone: TimeZone) => void;
```

Comment:
Wonder if the input should support time zone change by default. Is this something that's needed in your use case? It feels to me like the time zone change feature is intentional rather than default, and possibility to change it was drive by certain needs for quick access in dashboard.

Reply:
Yeah, makes sense to make it optional.